pkg/provider/pulumi/azure/serviceprinciple.go

```diff
@@ -40,10 +40,30 @@ func newServicePrincipal(ctx *pulumi.Context, name string, args *ServicePrincipa
 	if err != nil {
 		return nil, err
 	}
+	current, err := azuread.GetClientConfig(ctx, nil)
+	if err != nil {
+		return nil, err
+	}
 
 	// create an application per service principal
+	appRoleId := pulumi.String("4962773b-9cdb-44cf-a8bf-237846a00ab7")
```

Comment:
Is it better that this (line 64 as well) is a new uuid each time?

Reply:
From my understanding, that's a specific uuid which points to the `AzureEventGridSecureWebhookSubscriber` Role

Reply:
I see, sounds good!